### PR TITLE
Fix/add button handle

### PR DESCRIPTION
**Pull request by ancasolot**
#### What did you change? \*

Added a CSS handle to NewAddress button

#### Why? \*

To allow specific style customization to that element.

#### How to test it? \*

https://csshandler--vtexromania.myvtex.com/account#/addresses

#### Related to / Depends on?

https://gitlab.com/vtexro/1st-party-apps/support/-/issues/11

#### Types of changes \*

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical improvements

### CODE REVIEW

File: react/.eslintrc [hunk 1]
```diff
@@ -4,5 +4,9 @@
     "browser": true,
     "es6": true,
     "jest": true
+  },
+  "rules": {
+    "@typescript-eslint/camelcase": "off",
+    "@typescript-eslint/naming-convention": "warn"
```
Comment (salesfelipe) [inline react/.eslintrc:10]: Why did you change this?

Comment (ancasolot) [reply react/.eslintrc:10]: I changed it because the github pipeline was failing in multiple files that I didn't change with camelcase errors and I assumed camelcase should not be taken into consideration

**Review by salesfelipe — APPROVED** [review]
